fem/intrules.cpp

```diff
@@ -1794,7 +1794,7 @@ IntegrationRule *IntegrationRules::PyramidIntegrationRule(int Order)
    // rule for a cube which seems to be adequate for now. When we
    // implement high order finite elements for pyramids we should
```

Comment:
The comment should be updated since MFEM now has high-order elements for pyramids. I wish I knew a better integration rule to suggest